index.js

```diff
@@ -137,9 +144,11 @@ function getServiceBuilderFromService(baseOptions = {}) {
   return serviceBuilder(this[MICROSERVICE_GATEWAY_SERVICE_NAME], {}, baseOptions)
 }
 
+// TODO: test this
 function getMiaHeaders() {
   return {
     [this.USERID_HEADER_KEY]: this.getUserId(),
+    [this.USER_PROPERTIES_HEADER_KEY]: JSON.stringify(this.getUserProperties()),
```

Comment:
parse? with try / catch

Reply:
Nope, `stringify` should be correct, as it returns the original value that was in the headers. Actually I didn't fully understand the meaning of this function `getMiaHeaders`.